### PR TITLE
Reduce amount of logging at INFO level.

### PR DESCRIPTION
**Pull request by erikjohnston**
A lot of the things we log at INFO are now a bit superfluous, so lets
make them DEBUG logs to reduce the amount we log by default.

---

The big thing left is to reduce the number of log lines for outbound requests, but that is a bit more complicated so I've left that for now. We also might want to consider not logging GC gen 1 and 2 collections (potentially replacing with logging if they take a long time?)

Most of these were found by looking at jki.re logs and sytest logs.

### CODE REVIEW

File: synapse/http/site.py [hunk 1]
```diff
@@ -225,7 +225,7 @@ def _started_processing(self, servlet_name):
             self.start_time, name=servlet_name, method=self.get_method()
         )
 
-        self.site.access_logger.info(
+        self.site.access_logger.debug(
```
Comment (babolivier) [inline synapse/http/site.py]: I'd expect inbound traffic to be logged at `INFO` but ymmv.

File: changelog.d/6862.misc [hunk 2]
```diff
@@ -0,0 +1 @@
+Reduce amount we log at INFO level.
```
Comment (babolivier) [inline changelog.d/6862.misc]:
```suggestion
Reduce amount we log at `INFO` level.
```

File: synapse/http/site.py (hunk 1 above)
Comment (richvdh) [reply synapse/http/site.py]: we log when it completes at INFO. I think this is the right choice.

**Comment by erikjohnston** [comment]
> @andrewshadura had a few other suggestions at https://matrix.to/#/!cApDlsILLZkmAyVoxe:matrix.org/$1579358078840786DlSAb:matrix.org?via=matrix.org&via=sw1v.org&via=gruenhage.xyz etc.

Aha, thanks, I'll have a look

File: synapse/http/site.py (hunk 1 above)
Comment (babolivier) [reply synapse/http/site.py]: Fair enough.

**Pull request merged**